Documentation/ceph-pool-crd.md

```diff
@@ -84,6 +84,25 @@ When creating an erasure-coded pool, it is highly recommended to create the pool
 * `deviceClass`: Sets up the CRUSH rule for the pool to distribute data only on the specified device class. If left empty or unspecified, the pool will use the cluster's default CRUSH root, which usually distributes data over all OSDs, regardless of their class.
 * `crushRoot`: The root in the crush map to be used by the pool. If left empty or unspecified, the default root will be used. Creating a crush hierarchy for the OSDs currently requires the Rook toolbox to run the Ceph tools described [here](http://docs.ceph.com/docs/master/rados/operations/crush-map/#modifying-the-crush-map).
 * `compressionMode`: Sets up the pool for inline compression when using a Bluestore OSD. If left unspecified does not setup any compression mode for the pool. Values supported are the same as Bluestore inline compression [modes](https://docs.ceph.com/docs/master/rados/configuration/bluestore-config-ref/#inline-compression), such as `none`, `passive`, `aggressive`, and `force`.
+* `poolProperties`: Sets any [property](https://docs.ceph.com/docs/master/rados/operations/pools/#set-pool-values) listed to the given pool
+
+### Add specific pool properties
+
+With `poolProperties` you can set any pool property:
+
+```yaml
+spec:
+  poolProperties:
+    <name of the property>: <property value>
+```
+
+For instance:
+
+```yaml
+spec:
+  poolProperties:
+    pg_autoscale_mode: on
```

Comment:
We don't need pg_autoscale_mode as an example since it's set automatically, right?

Reply:
It's just an example but I can use another prop